### ISSUE TITLE
Possibility to also add  "normal" DNS services which provide an API.

### ISSUE THREAD

**Issue by l00mi** [troglobit/inadyn#182]
E.g. Cloudflare can be used as DDNS by just using the API which they provide.

**Comment by ysc3839** [comment]
Yes. For example: https://github.com/troglobit/inadyn/blob/b9d9196cbc435353cd79c8233f4623059f8e6983/plugins/cloudxns.c#L1

**Comment by troglobit** [comment]
Pull requests are most welcome, as usual.

**Comment by l00mi** [comment]
Perfect, thank you for the pointers. Will give it a try.

**Comment by ysc3839** [comment]
https://api.cloudflare.com/#dns-records-for-a-zone-update-dns-record

**Comment by ysc3839** [comment]
Cloudflare API provides search function so we can get zone/record ID easily.
